office-management-app/src/main/java/saadow/office_management_app/controller/OfficeController.java

```diff
@@ -30,19 +31,12 @@
 
 
 	@Autowired
-	private OfficeService officeService;
+	private OfficeService officeService = new OfficeServiceImpl();
```

Comment:
What is the reason to use @Autowired and new at the same time?

Reply:
Yes, it critical